README.md

```diff
@@ -23,14 +23,13 @@ Install the addon.
 Import the mixins in a component and use the events or the handlers.
 
 ```javascript
-import Ember from 'ember';
+import Component from '@ember/component';
```

Comment:
From what version of Ember is this way of importing modules supported? I see from Travis that it's at least 2.8 but maybe the earliest supported version should be mentioned somewhere in the README.